content/java/developer-guide/symbology/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Symbology"
+date: 2017-07-13T23:24:01-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: "java"
+projectname: "WorldWind Java"
+layout: "project"
+---
+
+## Symbology
+
+The WorldWind API enables applications to display common symbology sets. Currently supported is [MIL-STD-2525](http://worldwind.arc.nasa.gov/milstd2525c/Mil-STD-2525C.pdf).
```

Comment:
Given these straightforward list ordering options, lets take this up as a separate pull request. For the state we need the site in at the moment, it's good enough as-is.